pca-ssm/cfn/ssm.template

```diff
@@ -105,7 +105,7 @@ Parameters:
       <br><transcript>
       <br>{transcript}
       <br></transcript>
-      <br>Assistant: Here is the JSON object with the answers to the questions: 
```

Comment:
Same comment as above.. Missing a <br> here to match the Anthropic prompt docs `\n\nAssistant:`